domains/vihanga.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Share about my projects",
+    "repo": "https://github.com/vihangaashin/vihangaashin.github.io",
+    "owner": {
+        "username": "vihangaashin",
+        "email": "vihangapubg44@gmail.com",
+        "twitter": "vihangayt"
+    },
+    "record": {
+        "CNAME": "vihangaashin.github.io"
```

Comment:
Additional content should be displayed here.

Reply:
What??